packages/chain-events/src/chains/cosmos/filters/enricher.ts

```diff
@@ -46,10 +42,10 @@ export async function Enrich(
             typeUrl: proposal.content.typeUrl,
             value: Buffer.from(proposal.content.value).toString('hex'),
           },
-          submitTime: dateToUnix(proposal.submitTime),
-          depositEndTime: dateToUnix(proposal.depositEndTime),
-          votingStartTime: dateToUnix(proposal.votingStartTime),
-          votingEndTime: dateToUnix(proposal.votingEndTime),
+          submitTime: proposal.submitTime.seconds.toNumber(),
+          depositEndTime: proposal.depositEndTime.seconds.toNumber(),
+          votingStartTime: proposal.votingStartTime.seconds.toNumber(),
+          votingEndTime: proposal.votingEndTime.seconds.toNumber(),
```

Comment:
Change from cosmJS bump (.seconds):

>(property) Timestamp.seconds: Long.Long
Represents seconds of UTC time since Unix epoch 1970-01-01T00:00:00Z. Must be from 0001-01-01T00:00:00Z to 9999-12-31T23:59:59Z inclusive.